### PR TITLE
Troubleshooting network problems: use a more reliable URL, compliance.

### PR DESCRIPTION
**Pull request by themr0c**
* Use a more reliable URL. 
* Compliance with style guide, modular docs, minimalism.
* Fix vale checks: use che-docs container, validate complete new and changed files as validating the diffs doesn't work.

### PR Checklist

As the author of this Pull Request I made sure that:

- [x] `vale` has been run successfully against the PR branch
- [x] Link checker has been run successfully against the PR branch
- [x] Documentation describes a scenario that is already covered by QE tests, otherwise an issue has been created and acknowledged by Che QE team
- [x] Changed article references are updated where they are used (or a redirect has been set up on the docs side):
    - [x] Dashboard [branding.json](https://github.com/eclipse/che-dashboard/blob/master/src/components/branding/branding.json)
    - [x] Chectl [constants.ts](https://github.com/che-incubator/chectl/blob/master/src/constants.ts)

### CODE REVIEW

**Review by benoitf — APPROVED** [review]

File: modules/end-user-guide/partials/proc_troubleshooting-network-problems.adoc [hunk 1]
```diff
@@ -3,70 +3,25 @@
...
-+
-NOTE: Using an unsupported web browser causes a connection interruption, followed by a warning message.
 
 .Procedure
 
-. Check browser support:
-.. Check that the WebSocket protocol is enabled using a link:https://www.websocket.org/echo.html[realtime web test] in one of the supported browsers.
-+
-If the problem is not resolved, follow with the next step.
+. Verify the browser supports the WebSocket protocol. See: link:https://www.google.com/search?q=websocket+test[Searching a websocket test]
```
Comment (themr0c) [inline modules/end-user-guide/partials/proc_troubleshooting-network-problems.adoc]:
```suggestion
. Verify the browser supports the WebSocket protocol. See: link:https://www.google.com/search?q=websocket+test[Searching a websocket test].
```

**Comment by themr0c** [comment]
Now looking at the vale errors. It seems looking at the diffs is causing weird results:

```
cat /tmp/vale/modules/end-user-guide/partials/proc_troubleshooting-network-problems.adoc 
This section describes how to prevent or resolve issues related to network policies. 
{prod-short} requires the availability of the WebSocket Secure (WSS) connections.
Secure WebSocket connections improve confidentiality and also reliability because they reduce the risk of interference by bad proxies.
* The WebSocket Secure (WSS) connections on port 443 must be available on the network. Firewall and proxy may need additional configuration.
* Use a supported web browser:
. Verify the browser supports the WebSocket protocol. See: link:https://www.google.com/search?q=websocket+test[Searching a websocket test].
. Verify firewalls settings: WebSocket Secure (WSS) connections on port 443 must be available.
. Verify proxy servers settings: the proxy transmits and intercepts WebSocket Secure (WSS) connections on port 443.
 ffloreth  …/gh/the/che-docs  vale /tmp/vale/modules/end-user-guide/partials/proc_troubleshooting-network-problems.adoc  /tmp/vale/modules/end-user-guide/partials/proc_troubleshooting-network-problems.adoc
 6:55  suggestion  Try to keep sentences less      IBM.SentenceLength     
                   than 25 words.                                         
 6:55  warning     Keep sentences short and to     CheDocs.SentenceLength 
                   the point                                              ✖ 0 errors, 1 warning and 1 suggestion in 1 file.
 ffloreth  …/gh/the/che-docs  vale modules/end-user-guide/partials/proc_troubleshooting-network-problems.adoc
✔ 0 errors, 0 warnings and 0 suggestions in 1 file.
```

**Comment by themr0c** [comment]
Trying out the Vale action: https://github.com/errata-ai/vale-action
I can see it *building* the vale container at each run, I don't believe we really want that.

File: modules/end-user-guide/partials/proc_troubleshooting-network-problems.adoc [hunk 2]
```diff
@@ -3,70 +3,24 @@
...
-.. Ask the system administrator to check if there is a proxy server or firewall that blocks WebSocket Secure (WSS) connections on port 443.
-+
-Possible required actions:
-+
-* Add an exception to the firewall.
-* Have the proxy intercept WebSocket connection.
+. Verify the browser supports the WebSocket protocol. See: link:https://www.google.com/search?q=websocket+test[Searching a websocket test].
 
-.Verification
+. Verify firewalls settings: WebSocket Secure (WSS) connections on port 443 must be available.
```
Comment (MichalMaler) [inline modules/end-user-guide/partials/proc_troubleshooting-network-problems.adoc:24]: Verify firewall settings: WebSocket Secure (WSS) connections on port 443 must be available.

File: modules/end-user-guide/partials/proc_troubleshooting-network-problems.adoc [hunk 3]
```diff
@@ -3,70 +3,24 @@
...
 
-Check that the WebSocket protocol is enabled using a link:https://www.websocket.org/echo.html[realtime web test] in one of the supported browsers.
+. Verify proxy servers settings: the proxy transmits and intercepts WebSocket Secure (WSS) connections on port 443.
```
Comment (MichalMaler) [inline modules/end-user-guide/partials/proc_troubleshooting-network-problems.adoc]: Verify proxy server settings: the proxy transmits and intercepts WebSocket Secure (WSS) connections on port 443.

File: modules/end-user-guide/partials/proc_troubleshooting-network-problems.adoc (hunk 2 above)
Comment (themr0c) [reply modules/end-user-guide/partials/proc_troubleshooting-network-problems.adoc:24]: What is the difference?

File: modules/end-user-guide/partials/proc_troubleshooting-network-problems.adoc (hunk 3 above)
Comment (themr0c) [reply modules/end-user-guide/partials/proc_troubleshooting-network-problems.adoc]: What is the difference?

File: modules/end-user-guide/partials/proc_troubleshooting-network-problems.adoc (hunk 2 above)
Comment (MichalMaler) [reply modules/end-user-guide/partials/proc_troubleshooting-network-problems.adoc:24]: firewalls/firewall

File: modules/end-user-guide/partials/proc_troubleshooting-network-problems.adoc (hunk 3 above)
Comment (MichalMaler) [reply modules/end-user-guide/partials/proc_troubleshooting-network-problems.adoc]: servers/server

File: modules/end-user-guide/partials/proc_troubleshooting-network-problems.adoc (hunk 2 above)
Comment (themr0c) [reply modules/end-user-guide/partials/proc_troubleshooting-network-problems.adoc:24]: Plural form is intentional, as any corporate environment will have at lest 2 firewalls for redundancy purpose.

File: modules/end-user-guide/partials/proc_troubleshooting-network-problems.adoc (hunk 3 above)
Comment (themr0c) [reply modules/end-user-guide/partials/proc_troubleshooting-network-problems.adoc]: Plural form is intentional, as any corporate environment will have at lest 2 firewalls for redundancy purpose.

**Review by MichalMaler — APPROVED** [review]

**Comment by themr0c** [comment]
@benoitf Requesting your review again, to validate the modification in the workflows.

File: .github/workflows/vale-on-pr.yaml [hunk 4]
```diff
@@ -0,0 +1,28 @@
+#
+# Copyright (c) 2020 Red Hat, Inc.
```
Comment (benoitf) [inline .github/workflows/vale-on-pr.yaml]: you can bump to 2021

File: .github/workflows/vale-on-push.yaml [hunk 5]
```diff
@@ -0,0 +1,26 @@
+#
+# Copyright (c) 2020 Red Hat, Inc.
```
Comment (benoitf) [inline .github/workflows/vale-on-push.yaml]: 2021

**Comment by themr0c** [comment]
Screenshot from 2021-02-03 15-07-26

File: .github/workflows/vale-on-push.yaml [hunk 6]
```diff
@@ -0,0 +1,26 @@
...
+# which is available at https://www.eclipse.org/legal/epl-2.0/
+#
+# SPDX-License-Identifier: EPL-2.0
+#
+
+name: vale-push-check
+on:
+  - push
+jobs:
+  vale:
```
Comment (benoitf) [inline .github/workflows/vale-on-push.yaml]: maybe use vale-all-content vs vale-diff for the PR check ?

File: .github/workflows/vale-on-pr.yaml (hunk 4 above)
Comment (themr0c) [inline .github/workflows/vale-on-pr.yaml]:
```suggestion
# Copyright (c) 2021 Red Hat, Inc.
```

**Review by benoitf — APPROVED** [review]

File: .github/workflows/vale-on-push.yaml (hunk 5 above)
Comment (themr0c) [inline .github/workflows/vale-on-push.yaml]:
```suggestion
# Copyright (c) 2021 Red Hat, Inc.
```

Comment (themr0c) [inline .github/workflows/vale-on-push.yaml]:
```suggestion
# Copyright (c) 2021 Red Hat, Inc.
```

File: .github/workflows/vale-on-push.yaml [hunk 7]
```diff
@@ -0,0 +1,26 @@
...
```
Comment (themr0c) [inline .github/workflows/vale-on-push.yaml]:
```suggestion
  vale-all-content:
```

File: .github/workflows/vale-on-pr.yaml [hunk 8]
```diff
@@ -0,0 +1,28 @@
...
+# which is available at https://www.eclipse.org/legal/epl-2.0/
+#
+# SPDX-License-Identifier: EPL-2.0
+#
+
+name: vale-pr-check
+on:
+  - pull_request
+jobs:
+  vale:
```
Comment (themr0c) [inline .github/workflows/vale-on-pr.yaml]:
```suggestion
  vale-diff:
```

**Pull request merged**